monitor/census.go

```diff
@@ -1141,6 +1141,8 @@ func SetTranscodersNumberAndLoad(load, capacity, number int) {
 	stats.Record(census.ctx, census.mTranscodersLoad.M(int64(load)))
 	stats.Record(census.ctx, census.mTranscodersCapacity.M(int64(capacity)))
 	stats.Record(census.ctx, census.mTranscodersNumber.M(int64(number)))
+	stats.Record(census.ctx, census.mWinningTicketsRecv.M(int64(number)))
```

Comment:
I'm not sure I understand it correctly. The `number` param in this function is the number of Transcoders. Why do we set the number of winning tickets to the number of Transcoders? 🤔

Reply:
Moved functions to 
// init metrics values
	SetTranscodersNumberAndLoad(0, 0, 0)

Been running with these changes in production since last week

Reply:
Good. could you remove it from here? I think the lines you added ☝️ are enough.